comms/performance-test/client/index.ts

```diff
@@ -0,0 +1,361 @@
+import { Peer } from "../../peer/src/Peer";
+import { randomBetween } from "decentraland-katalyst-utils/util";
+import { PeerMessageTypes } from "../../peer/src/messageTypes";
+import { Position3D, Quaternion } from "decentraland-katalyst-utils/Positions";
+import { PeerConfig } from "../../peer/src";
+import { PositionData, CommsMessage, ProfileData, ChatData } from "./protobuf/comms";
+import { util } from "../../peer/src/peerjs-server-connector/util";
+import { GlobalStats } from "../../peer/src/stats";
+import { Reader } from "protobufjs";
+
+function uuid(): string {
```

Comment:
We have a uuid library dependency already. Check the class `NameKeeper` on the content module